README.md

```diff
@@ -110,5 +110,37 @@ This function is deprecated. Use the `torch.nn.attention.sdpa_kernel` context ma
 Migration guide:
 Each boolean input parameter (defaulting to true unless specified) of `sdp_kernel` corresponds to a `SDPBackened`. If the input parameter is true, the corresponding backend should be added to the input list of `sdpa_kernel`.
 
+#### torch.chain_matmul
+
+This function is deprecated in favor of `torch.linalg.multi_dot`.
+
+Migration guide:
+`multi_dot` accepts a list of two or more tensors whereas `chain_multimul` accepted multiple tensors as input arguments. For migration, convert the multiple tensors in argument of  `chain_multimul` into a list of two or more tensors for `multi_dot`.
```

Comment:
This talks about `chain_multimul`, not `chain_matmul`

Reply:
Addresses changes in 34f8af9